internal/util/importutilv2/json/row_parser.go

```diff
@@ -401,6 +404,147 @@ func (r *rowParser) parseEntity(fieldID int64, obj any) (any, error) {
 	}
 }
 
+func (r *rowParser) parseNullableEntity(fieldID int64, obj any) (any, error) {
```

Comment:
This piece of code looks repetitive with `parseEntity`. Is it possible that we merge these two functions together?

Reply:
I write separately to make logic more clear. but it's ok to combine it. I will do it in next pr.